addons/survey/models/survey_survey.py

```diff
@@ -79,7 +87,8 @@ def default_get(self, fields_list):
         ('page_per_question', 'One page per question'),
         ('page_per_section', 'One page per section'),
         ('one_page', 'One page with all the questions')],
-        string="Pagination", required=True, default='page_per_question')
+        string="Pagination", required=True, default='page_per_question',
```

Comment:
Note that if it becomes a computed editable, we should not have a default anymore, as everything should be contained in the compute method.